### PR TITLE
fix hasSameNode to compare by key (instead of name)

### PR DESCRIPTION
**Pull request by mariakleiner**

### CODE REVIEW

File: pkg/Library/NodeGraph/NodesConnector.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ updateNodes({pipeline, selectedNode, nodeTypes, customInspectors, inspectorData,
 },
 
 hasSameNode(node, nodes) {
-  const nodeInNodes = nodes.find(n => n.name === node.name);
+  const nodeInNodes = nodes.find(n => n.name === node.key);
```
Comment (sjmiles) [inline pkg/Library/NodeGraph/NodesConnector.js:46]: key === key

**Review by sjmiles — APPROVED** [review]

File: pkg/Library/NodeGraph/NodesConnector.js (hunk 1 above)
Comment (sjmiles) [reply pkg/Library/NodeGraph/NodesConnector.js:46]: needs fixing

**Review by sjmiles — CHANGES_REQUESTED** [review]

File: pkg/Library/NodeGraph/NodesConnector.js (hunk 1 above)
Comment (mariakleiner) [reply pkg/Library/NodeGraph/NodesConnector.js]: duh, sorry :( fixed now

**Comment by mariakleiner** [comment]
fixed, ptal

**Review by sjmiles — APPROVED** [review]

**Pull request merged**